platform-configs/protectli-vp6670.robot

```diff
@@ -0,0 +1,20 @@
+*** Settings ***
+Resource    include/protectli-vp46xx.robot
```

Comment:
Should be 66xx.

Reply:
fixed in recent [commit](https://github.com/Dasharo/open-source-firmware-validation/pull/249/commits/2ba06485707f5a953641db1b24719078f25c34db#diff-b7aa45df0aed3cc0c6233b9af2f88ec9e6aa71eb0960168b64f4ae5843635834R2)